sparql/sparql11/aggregates/agg-groupconcat-5.srx

```diff
@@ -0,0 +1,7 @@
+PREFIX : <http://www.example.org/>
```

Comment:
Something probably went wrong here. Should be a results file.

Reply:
Sorry for that. It seems I got mixed up in my MRs and forgot to run this one with Oxigraph. Fixed and the tests pass with Oxigraph.